source/faqs.rst

```diff
@@ -373,3 +367,208 @@ The ``--no-builds`` argument completely removes the build number from the
 output, avoiding future errors when trying to rebuild the environment, and
 allowing the conda solver to identify the packages that can co-exist in the
 same environment.
+
+How are dependencies pinned to particular versions?
+---------------------------------------------------
+
+In some cases a recipe may need to pin the version of a dependency.
+A global set of default versions to pin against is shared with conda-forge and
+can be found `here <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`_.
+For new dependencies that are contained in conda-forge and not yet in this list,
+please update the list via a pull request.
+Local pinnings can be achieved by adding a file ``conda_build_config.yaml`` next
+to your ``meta.yaml``.
+
+To find out against which version you can pin a package, e.g. x.y.* or x.* please use `ABI-Laboratory <https://abi-laboratory.pro/tracker/>`_.
+
+What's the lifecycle of a bioconda package?
+-------------------------------------------
+
+- Submit a pull request with a new recipe or an updated recipe
+- Circle CI automatically builds and tests the changed recipe[s] using
```

Comment:
`Circle CI` would be used only for `linux-aarch64`, right ?
So here it should be used something general like `The CI`